src/change_stream/document.rs

```diff
@@ -0,0 +1,97 @@
+use crate::coll::Namespace;
```

Comment:
This file should have a comment on top with a description of the module (see `src/event/mod.rs` for an example).

Reply:
fixed